omnibus/config/software/datadog-logs-agent-win.rb

```diff
@@ -40,11 +40,11 @@
   end
 
   #command "go get github.com/Masterminds/glide", :env => env
-  command "glide install", :env => env
-  command "rake build", :env => env
+  command "inv deps", :env => env
+  command "inv logs.build", :env => env
   if windows?
-    copy "build/#{log_agent_binary}", "#{install_dir}/bin/agent"
+    copy "bin/logs/#{log_agent_binary}", "#{install_dir}/bin/agent/logs-agent"
```

Comment:
from @olivielpeau 

on Windows we probably want the installed binary to have the .exe extension